apps/earn-protocol/app/server-handlers/sdk/get-global-rebalances.ts

```
@@ -1,31 +1,39 @@
-    // initially sorted by timestamp since it combines data from independent subgraphs
-    .sort((a, b) => simpleSort({ a: a.timestamp, b: b.timestamp, direction: SortDirection.DESC }))
 
-  return {
-    rebalances: rebalancesList,
-    callDataTimestamp: Date.now(),
-  }
-}
+    // flatten the list
+    const rebalancesList = rebalancesByNetwork
+      .reduce<(typeof rebalancesByNetwork)[number]['rebalances']>(
+        (acc, { rebalances }) => [...acc, ...rebalances],
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)